Automation/SQL/azureDeploy.ps1

```diff
@@ -0,0 +1,24 @@
+param(
+        [string] [Parameter(Mandatory=$true)] $location,
+        [string] [Parameter(Mandatory=$true)] $resourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $serverName,
+        [string] [Parameter(Mandatory=$true)] $tenantId,          
+        [string] [Parameter(Mandatory=$true)] $applicationId,
+        [string] [Parameter(Mandatory=$true)] $secret,
+        [string] [Parameter(Mandatory=$true)] $subscriptionId
+)
+
+$pathToAzSqlTemplate = "./azureDeploy.json"
+   
+& "../login.ps1" $tenantId $applicationId $secret $subscriptionId
+
+New-AzResourceGroup -Name $resourceGroupName -Location "$location"
```

Comment:
add -Force to this so it does the default (updates rg) when the resource group already exists

Reply:
done